### PR TITLE
Mobile SDK v3.2.0 Release

### PR DESCRIPTION
**Pull request by bhariharan**

### CODE REVIEW

File: libs/SalesforceNetwork/SalesforceNetwork/SalesforceNetworkTests/Output/CSFChatterChatterGroupPageOutputTest.m [hunk 1]
```diff
@@ -0,0 +1,35 @@
+//
+//  CHConnectChatterGroupPageResponseTest.h
+//  ChatterSDK
+//
+//  Created automatically by Michael Nachbaur on 09/05/13.
+//  Copyright (c) 2012 Salesforce.com. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
```
Comment (khawkins) [inline libs/SalesforceNetwork/SalesforceNetwork/SalesforceNetworkTests/Output/CSFChatterChatterGroupPageOutputTest.m]: I thought all of these had been removed....

Comment (bhariharan) [inline libs/SalesforceNetwork/SalesforceNetwork/SalesforceNetworkTests/Output/CSFChatterChatterGroupPageOutputTest.m]: They had! I wonder how they came back. Oh god.

Comment (bhariharan) [inline libs/SalesforceNetwork/SalesforceNetwork/SalesforceNetworkTests/Output/CSFChatterChatterGroupPageOutputTest.m]: I'll fix it.

Comment (bhariharan) [inline libs/SalesforceNetwork/SalesforceNetwork/SalesforceNetworkTests/Output/CSFChatterChatterGroupPageOutputTest.m]: I don't understand how, but ```git``` is pulling them in on the merge.

Comment (bhariharan) [inline libs/SalesforceNetwork/SalesforceNetwork/SalesforceNetworkTests/Output/CSFChatterChatterGroupPageOutputTest.m]: Never mind. Figured it out. Thanks for catching this. I'll overwrite this.

**Pull request merged**